build-time-analytics/src/test/java/io/quarkus/ts/buildtimeanalytics/QuarkusCliDevModeIT.java

```diff
@@ -24,20 +24,17 @@
 import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.condition.DisabledOnOs;
-import org.junit.jupiter.api.condition.OS;
 
 import io.quarkus.test.bootstrap.QuarkusCliRestService;
 import io.quarkus.test.bootstrap.RestService;
 import io.quarkus.test.scenarios.QuarkusScenario;
 import io.quarkus.test.scenarios.annotations.DisabledOnNative;
 
-@DisabledOnOs(OS.WINDOWS) // TODO mvavrik: must only run when Quarkus CLI and Docker are available
 @Tag("QUARKUS-2812")
 @Tag("quarkus-cli")
 @QuarkusScenario
 @DisabledOnNative(reason = "Only for JVM verification")
-public class DevModeIT extends AbstractAnalyticsIT {
+public class QuarkusCliDevModeIT extends AbstractAnalyticsIT {
```

Comment:
And why don't we add `@EnabledWhenLinuxContainersAvailable` here?

Reply:
Sure, if we don't go with the renaming, we can add the annotation and it will do the same job.